src/xunit.analyzers/Descriptors.cs

```diff
@@ -220,6 +220,16 @@ internal static class Descriptors
             "Do not use {0} to check for collection size.",
             Categories.Assertions, DiagnosticSeverity.Warning, isEnabledByDefault: true);
 
+        internal static DiagnosticDescriptor X2014_AssertThrowsShouldNotBeUsedForAsyncThrowsCheck { get; } = new DiagnosticDescriptor("xUnit2014",
+            "Do not use throws check to check for asynchronously thrown exception",
+            "Do not use {0} to check for asynchronously thrown exceptions.",
+            Categories.Assertions, DiagnosticSeverity.Warning, isEnabledByDefault: true);
```

Comment:
This should be Error, since not addressing the problem will cause the test to fail at runtime.